app/views/shared/text/_subtitle.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class="subtitle">
```

Comment:
i already gave it a class of tag name.. i don't think we need a component for that? because its on the front-page and not bothering with the searches going on

Reply:
i mean tag-line

Reply:
included so we can use it if we want :)